### ISSUE TITLE
[Bug] LinearMean bias fix

### ISSUE THREAD

**Issue by stanbiryukov** [cornellius-gp/gpytorch#873]
# 🐛 Bug

The linear mean kernel fails when bias is not set to True.

## To reproduce
Example google colab notebook with error and solution:
https://colab.research.google.com/drive/19mF1xPoH7r7DvywodQ34do9IjAA_lfDd

** Code snippet to reproduce **
```python
from gpytorch.means import Mean

## Fix by registering bias as None first
class LBMean(Mean):
    def __init__(self, input_size, batch_shape=torch.Size(), bias=True):
        super().__init__()
        self.register_parameter(name='weights',
                                parameter=torch.nn.Parameter(torch.randn(*batch_shape, input_size, 1)))
        self.register_parameter('bias', None)
        if bias:
            self.register_parameter(name='bias', parameter=torch.nn.Parameter(torch.randn(*batch_shape, 1)))
        
    def forward(self, x):
        res = x.matmul(self.weights).squeeze(-1)
        if self.bias is not None:
            res = res + self.bias
        return res

class GPRegressionModel(gpytorch.models.ExactGP):
    def __init__(self, train_x, train_y, likelihood):
        super(GPRegressionModel, self).__init__(train_x, train_y, likelihood)
        self.mean_module = LinearMean(input_size=train_x.shape[-1], bias=None) # doesn't work with no bias
        # self.mean_module = LBMean(input_size=train_x.shape[-1], bias=None) # this works
        self.base_covar_module = ScaleKernel(RBFKernel())
        self.covar_module = InducingPointKernel(self.base_covar_module, inducing_points=train_x[:500, :], likelihood=likelihood)

    def forward(self, x):
        mean_x = self.mean_module(x)
        covar_x = self.covar_module(x)
        return MultivariateNormal(mean_x, covar_x)
```

** Stack trace/error message **
```

AttributeError: 'LinearMean' object has no attribute 'bias'

```

## Expected Behavior
Successful training

## System information

**Please complete the following information:**
- 
0.3.5
- 
1.1.0
- 
posix.uname_result(sysname='Linux', nodename='c02afd15d7f2', release='4.14.137+', version='#1 SMP Thu Aug 8 02:47:02 PDT 2019', machine='x86_64')

## Additional context
We just need to register the bias parameter as None first for it to work in the forward pass syntax.  See LBMean definition above.

**Comment by KeAWang** [comment]
Thanks! This was my mistake. The fix will be merged soon.

**Comment by jacobrgardner** [comment]
Closed by #874
